graphql/src/execution/execution.rs

```diff
@@ -183,13 +172,7 @@ pub fn prefetch(
     ctx: &ExecutionContext<impl Resolver>,
     selection_set: &q::SelectionSet,
 ) -> Result<Option<q::Value>, Vec<QueryExecutionError>> {
-    // Allow turning prefetch off as a safety valve. Once we are confident
-    // prefetching contains no more bugs, we should remove this env variable
-    if *NO_PREFETCH {
-        Ok(None)
-    } else {
-        ctx.resolver.prefetch(&ctx, selection_set)
-    }
+    ctx.resolver.prefetch(&ctx, selection_set)
```

Comment:
Now that this is a one liner, you could just inline it in the one place where it is called.